src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsLookAndFeel.java

```diff
@@ -184,9 +183,7 @@ public void initialize() {
         // performance and compatibility issues, so allow this feature
         // to be switched off either at runtime or programmatically
         //
-        @SuppressWarnings("removal")
-        String systemFonts = java.security.AccessController.doPrivileged(
-               new GetPropertyAction("swing.useSystemFontSettings"));
+        String systemFonts = System.getProperty("swing.useSystemFontSettings");
```

Comment:
This file has a now-unused import of `GetPropertyAction`.

Reply:
removed